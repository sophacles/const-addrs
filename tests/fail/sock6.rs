use const_addrs::sock6;

fn main() {
    let _ = sock6!("2001:db8::32::23:22");
}
