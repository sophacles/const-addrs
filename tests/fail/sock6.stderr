error: invalid IPv6 socket address syntax
 --> tests/fail/sock6.rs:4:20
  |
4 |     let _ = sock6!("2001:db8::32::23:22");
  |                    ^^^^^^^^^^^^^^^^^^^^^
